buildlib/tools/builds.sh

```diff
@@ -236,10 +241,23 @@ build_cuda() {
 		echo "==== Running test_link_map with cuda build but no cuda module ===="
 		env UCX_HANDLE_ERRORS=bt ./test/apps/test_link_map
 	else
-		echo "==== Not building with cuda flags ===="
+                echo "==== Not building with cuda flags ===="
```

Comment:
why changed?

Reply:
that is a mistake, will fix. I had it briefly changed to an azure_log_warning for making debugging easier, since the cuda cases are now not compiled anymore in the rocm image. Will fix it.